cncserver.js

```diff
@@ -1125,6 +1126,18 @@ function serialPortReadyCallback() {
       return;
     }
 
+    // Setting the value of the power to the pen
+    if (typeof inPen.power !== "undefined") {
+      powers = botConf.get('penpower');
+      if(typeof powers === "undefined") { // We have no super powers
+        powers = {min: 0, max: 0};  //Set the powers to zero
+      }
+
+      run('custom', cmdstr('penpower', {p: Math.round(inPen.power * powers.max) + Number(powers.min)}));

```

Comment:
I'm not totally sold on this use of custom but... we're using it for all the "settings", and we're properly using `cmdstr()` here so, I guess I'll allow it.